### ISSUE TITLE
Enable Groups to publish to calendar

### ISSUE THREAD

**Issue by hagbardc** [IndivisibleOnline/Indivisible-Plugin#1]
####  Is there a signoff process required, or if an event is added by a group coordinator that's assumed as OK?

**Comment by howardstevens** [comment]
I am making good progress on this issue

**Comment by imlindaryan** [comment]
So - groups can now make their events public to the main site calendar with Admin approval.  However, can they publish only to other IW groups?

**Comment by howardstevens** [comment]
No, they can't yet make their events accessible to other groups only.
